ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/pre-external-lb/provision/cucushift-installer-rehearse-vsphere-ipi-pre-external-lb-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-pre-external-lb-provision
+  steps:
+  - ref: ipi-conf-vsphere-check
+  - ref: ipi-conf-vsphere-lb-external
+  - ref: ipi-conf-vsphere-dns
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-vsphere
+  - ref: ipi-conf-feature-set 
```

Comment:
The pre external lb is a TP feature?

Reply:
+1, didn't find any description as it is TP feature in [SPLAT-860](https://issues.redhat.com/browse/SPLAT-860)

Reply:
I add that step because i see this step in ipi-conf-vsphere-lb-external-chain.yaml .
cat ipi-conf-vsphere-lb-external-chain.yaml 
chain:
  as: ipi-conf-vsphere-lb-external
  steps:
  - ref: ipi-conf-vsphere-check
  - ref: ipi-conf-vsphere-lb-external
  - ref: ipi-conf-vsphere-dns
  - ref: ipi-conf
  - ref: ipi-conf-telemetry
  - ref: ipi-conf-vsphere
  - ref: ipi-conf-feature-set

Reply:
I do not think it is right things for `ipi-conf-vsphere-lb-external-chain.yaml`, I think we can update the chain yaml to drop `ipi-conf-feature-set` ref, and call the `chain: ipi-conf-vsphere-lb-external` directly here.

Reply:
ok, thanks for your advice.